crates/isograph_api/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "isograph_api"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+isograph_schema = { path = "../isograph_schema" }
+lazy_static = "1.4"
+regex = "1.6.0"
+colored = "2.0.4"
+serde = "1.0.190"
+serde_json = "1.0.108"
```

Comment:
serde and serde_json are maybe not needed in `isograph_cli` now

Reply:
Fixed with [`1236591` (#26)](https://github.com/isographlabs/isograph/pull/26/commits/12365914ff55d6c3839f90efabf8498be374ee1a)